src/ILCompiler.Compiler/src/Compiler/CompilerTypeSystemContext.cs

```diff
@@ -294,7 +294,7 @@ public override VirtualMethodAlgorithm GetVirtualMethodAlgorithmForType(TypeDesc
             return _virtualMethodAlgorithm;
         }
 
-        protected override IEnumerable<MethodDesc> GetAllMethods(TypeDesc type)
+        public override IEnumerable<MethodDesc> GetAllMethods(TypeDesc type)
```

Comment:
Same question as above - why is this public? Only type system should consume it.